docs/releases/1.0.0.md

```diff
@@ -0,0 +1,75 @@
+# Orchard Core Commerce 1.0.0
```

Comment:
The content of the file is copy-pasted right? (It's not a problem, just asking if I need to check this in detail.)

Reply:
Yes, it's copied from the GH Releases page, and then cleaned up according to the markdownlint warnings.
I considered creating a GH workflow that generates these release docs using the GH API, but it didn't seem worth it because these NuGet releases will be fairly rare.